### ISSUE TITLE
Oplog-Manager: Incorrect behaviour of  nested fields update  

### ISSUE THREAD

**Issue by jacquesdufeu** [mongodb-labs/mongo-connector#388]
There is an issue when fields parameter is  used  and an update is done on a nested field.
 For  example, if you keep only the "a" field of the document (fields parameter set to a)  and you update the nested "a.b" field  in mongoDB  ( db.doc.update(    { $set: { "a.b": "zzz" } }) ) , this update will not be applied to ES index

**Comment by aherlihy** [comment]
Hello! What version of the database are you using?

Also, we just released mongo-connector 2.2 today which requires PyMongo version >= 2.9. Does this problem persist with the new release?

**Comment by jacquesdufeu** [comment]
We are using mongo 2.6 
I didn't try with this new release  and I will not because unfortunately, I no longer work on this project.  . In fact I made a fork of this project on our own gitLab to fix that issue, . I change the pop_excluded_fields function in the oplog_manager.py file in order to keep update on a nested field if his root field  has to be indexed. 

Jacques
  

      De : aherlihy <notifications@github.com>
 À : mongodb-labs/mongo-connector <mongo-connector@noreply.github.com> 
Cc : jacquesdufeu <jacquesdufeu@yahoo.fr>
 Envoyé le : Lundi 15 février 2016 16h26
 Objet : Re: [mongo-connector] Oplog-Manager: Incorrect behaviour of nested fields update (#388)
   
Hello! What version of the database are you using?Also, we just released mongo-connector 2.2 today which requires PyMongo version >= 2.9. Does this problem persist with the new release?—
